src/utils/DataHelper.js

```diff
@@ -0,0 +1,17 @@
+export const isPeselValid = pesel => {
+  if (typeof pesel === "undefined" || !/^[0-9]{11}$/.test(pesel))
+    return "Type PESEL";
+
+  const weights = [1, 3, 7, 9, 1, 3, 7, 9, 1, 3];
+  let sum = 0;
+
+  weights.forEach((weight, index) => {
+    sum += weight * pesel.substring(index, index + 1);
+  });
+
+  const checkSum = sum % 10 === 0 ? 0 : 10 - (sum % 10);
+
+  if (checkSum !== +pesel.substring(10, 11)) return "Invalid PESEL";
+
+  return undefined;
```

Comment:
Is there required? I guess if a function has no return at the end by default return `undefined`

Reply:
Yes, you are right. If function has no `return`, it returns `undefined` by default.

`return` has been removed from this function.